tools/Ecoregionalization_workflow/indval.xml

```diff
@@ -0,0 +1,108 @@
+<tool id="indval" name="Indicspecies" version="0.1.0+galaxy0">
+    <description>to identify indicator species within clusters/groups</description>
+    <requirements>
+      <requirement type="package" version="4.4.2">r-base</requirement>
+      <requirement type="package" version="1.1.4">r-dplyr</requirement>
+      <requirement type="package" version="1.7.15">r-indicspecies</requirement>
+      <requirement type="package" version="0.9_4">r-permute</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    Rscript '$__tool_directory__/indval.R' '$clus' '$occ' '$name_species' '$sign'
+    ]]></command>
+    <inputs>
+      <param name="occ" type="data" format="tabular" label="Input your Occurences file (tabular format only)" help="See example below"/>
+      <param name="clus" type="data" format="tabular" label="Input your cluster file (tabular format only)" help="See example below"/>
+      <param name="name_species" type="data_column" label="Choose column(s) where your species are in your occurrence data file." data_ref="occ" multiple="true" use_header_names="true"/>
+      <param name="sign" type="boolean" label="Display only significant results (p-value inferior to 0.05)" checked="true"/>
+    </inputs>
+    <outputs>
+      <data name="output" from_work_dir="indval.txt" format="txt" label="Indicator values"/>
+    </outputs>
+    <tests>
+      <test>
+        <param name="occ" value="Merged_table.tabular"/>
+        <param name="clus" value="Cluster_points.tabular"/>
+        <param name="name_species" value="3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73"/>
+        <param name="sign" value="true"/>
+        <output name="output">
+           <assert_contents>
+              <has_size value="4175" delta="50"/>
```

Comment:
please use better asserts, e.g. asserting the number of lines or columns etc ...

Reply:
Thanks a lot for your review, I'll do that.